docs/podman-create.1.md

```diff
@@ -476,7 +496,8 @@ standard input.
    Without this argument the command will be run as root in the container.
 
 **--userns**=""
-   Set the usernamespace mode for the container when `userns-remap` option is enabled.
+   Set the usernamespace mode for the container when `userns-remap` option is enabled. The use of userns is disabled by default, unless you turn on --uidmap or --gidmap.
```

Comment:
Where is the userns-remap option?  A location here would be nice and/or a man page link in the SEE ALSO section.

Reply:
Should be removed we don't expose that option to the user.

For podman --userns=host is default --userns=contaier:foobar is probably not enabled.
and should conflict with --uidmap and --gidmap